src/Api/Controller/ListUsersController.php

```diff
@@ -85,12 +85,13 @@ protected function data(ServerRequestInterface $request, Document $document)
 
         $filters = $this->extractFilter($request);
         $sort = $this->extractSort($request);
+        $sortIsDefault = $this->sortIsDefault($request);
```

Comment:
can we rename `sortIsDefault` to `isDefaultSorting` or maybe `isSortingByDefault`.

Reply:
Mixed feelings about this one. Any use of sort**ing** describes whether or not **any** sort is being applied. But this variable should describe whether the sort currently being used (whatever that is, could be no sort at all) is the controller's default sort.

Alternatively, we could use `userRequestedSort` or `userSpecifiedSort` and invert everything, but that sounds very verbose.

Reply:
True, we only want to know if we're falling back to the default sort.

Though I was more looking for just a `isSomething` naming, since that's the usual convention for boolean variables/methods where possible. But `isDefaultSort` might be weird.

Let's keep it as is then!